arraycontainer.go

```diff
@@ -439,6 +439,7 @@ func (ac *arrayContainer) lazyorArray(value2 *arrayContainer) container {
 }
 
 func (ac *arrayContainer) and(a container) container {
+	//p("ac.and() called")
```

Comment:
In fact I already felt the need to maintain a separate fork because the comments got stripped out of other parts. It really makes maintenance so much easier to have them still available.